tools/build_config/build.ps1

```diff
@@ -188,6 +192,43 @@ function Invoke-Package {
   }
 }
 
+function Invoke-Build-Docs {
+  (Get-Content "./build/CPackConfig.cmake" | Where-Object {$_ -match 'CPACK_PACKAGE_FILE_NAME'}) -match '.*"Horace-([^"]+)".*'
+  $build_id = $Matches[1]
+  (Get-Content ./documentation/user_docs/docs/conf.py) | Foreach-Object {$_ -replace 'release = .*', "release = '${build_id}'"} | Set-Content  ./documentation/user_docs/docs/conf.py
+
+  ./documentation/user_docs/make.bat html
+  ./documentation/user_docs/make.bat html
+  # Undo change to allow checkout
+  git checkout ./documentation/user_docs/docs/conf.py
+
+  Foreach ($f in Get-ChildItem -Path './documentation/user_docs/build/html' -Filter *.html) {
+      (Get-Content "./documentation/user_docs/build/html/$f") | Where-Object {$_ -notmatch '\\[NULL\\]'} | Set-Content "./documentation/user_docs/build/html/$f"
```

Comment:
This line is quite long, can we split the lines on the pipes? The line continuation for PowerShell is a backtick